package.json

```diff
@@ -123,20 +127,21 @@
     "@types/react-redux": "^6.0.1",
     "@types/socket.io-client": "^1.4.32",
     "babel-eslint": "^10.0.3",
-    "babel-jest": "23.0.1",
+    "babel-jest": "^24.9.0",
     "babel-preset-react-native": "4.0.0",
-    "eslint": "^6.7.1",
+    "eslint": "^6.5.1",
```

Comment:
Why downgrade it? Or is thius the one that came builtin with RN?

Reply:
Yeah, this is the one that comes with React Native 0.61.5